### DISCUSSION TITLE
[Support]: Frigate Nginx Proxy Manager constant ping for entitelments

### DISCUSSION THREAD

**Discussion by gacekk** [blakeblackshear/frigate#13060] [General Support] [answered]
### Describe the problem you are having

Hi,

I managed to set up NPM to ping for password, however, when I first enter the password I would be getting another promptfor password repeatedly, that I can just cancell and continue viewing Frigate

### Version

14 but previous version as well

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

n/a

**Comment by NickM-27** [comment]
Are you using the built in authentication?

**Reply by gacekk** [reply]
Was there one introduced in v14?  Could not find that option on how to set it up. I have a user created but I don’t get pinged for password when accessing frigate

**Reply by NickM-27** [reply, marked as answer]
https://docs.frigate.video/configuration/authentication

**Reply by gacekk** [reply]
Thank you
